test/Performance/setup_env_unix.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+set -e
+if [[ ("$1" = "Ubuntu16" || "$1" = "RedHat7" || "$1" = "Sierra") ]]; then
+    PLATFORM=$1
+else
+    echo "First argument must be one of Ubuntu16, RedHat7, Sierra. Exiting..."
+    exit 1
+fi
+if [[ "$2" = "" ]]; then
+    echo "Second argument must be PHP version in format of 7.x.x.Exiting..."
+    exit
+else
+    PHP_VERSION=$2
+fi
+if [[ ("$3" != "nts" && "$3" != "ts") ]]; then
+    echo "Thrid argument must be either nts or ts. Exiting..."
+    exit 1
+else
+    PHP_THREAD=$3
+fi
+if [[ (! -d "$4") || (! -d $4/sqlsrv) || (! -d $4/pdo_sqlsrv) || (! -d $4/shared) ]]; then
+    echo "Fourth argument must be path to source folder.Path not found.Exiting..."
+    exit 1
+else
+    DRIVER_SOURCE_PATH=$4
+fi
+rm -rf env_setup.log
+touch env_setup.log
+if [ $PLATFORM = "Ubuntu16" ]; then
+    echo "Update..."
+    yes | sudo dpkg --configure -a >> env_setup.log 2>&1
+    yes | sudo apt-get update >> env_setup.log 2>&1
+    echo "Installing git, zip, curl, libxml, autoconf, openssl, python3, pip3..."
+    yes | sudo apt-get install git zip curl autoconf libxml2-dev libssl-dev pkg-config python3 python3-pip >> env_setup.log 2>&1  
+    echo "OK"
+    echo "Installing MSODBCSQL..."
+    curl -s https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+    curl -s https://packages.microsoft.com/config/ubuntu/16.04/prod.list > /etc/apt/sources.list.d/mssql-release.list
+    yes | sudo apt-get update >> env_setup.log 2>&1
+    yes | sudo ACCEPT_EULA=Y apt-get install msodbcsql >> env_setup.log 2>&1
+    yes | sudo apt-get install -qq unixodbc-dev >> env_setup.log 2>&1
+    echo "Installing pyodbc"
+    pip3 install --upgrade pip >> env_setup.log 2>&1
+    pip3 install pyodbc >> env_setup.log 2>&1
+    echo "OK"
+elif [ $PLATFORM = "RedHat7" ]; then
+    echo "Update..."
+    yes | sudo yum update >> env_setup.log 2>&1
+    echo "Enabling EPEL repo..."
+    yes | sudo yum install epel-release -y >> env_setup.log 2>&1
+    echo "OK"
+    echo "Installing python34-setuptools..."
+    yes | sudo yum install python34-setuptools -y >> env_setup.log 2>&1
+    echo "OK"
+    echo "Installing gcc, git, zip libxml, openssl, EPEL, python3, pip3..."
```

Comment:
remove EPEL and pip3 as they are install at earlier and later steps respectively